py/build.gradle

```diff
@@ -3,6 +3,13 @@ apply from: "$rootDir/gradle/utils.gradle"
 ext {
     FS = File.separator
     FPS = File.pathSeparator
+
+    def command = ["python", "py/pythonVersion.py"]
```

Comment:
Why do you need separated python file? You can invoke python code directly no?

I am asking since it is inconsistent with code above.